tests/test_python_direct.py

```diff
@@ -38,3 +38,22 @@ def test_python_metta_dialog():
     ''')
     assert a('(user "Echo")').content == [ValueAtom('user Echo')]
     assert a('(user "Recall")').content == [ValueAtom('user Echo\nassistant user Echo')]
+
+def test_stream_response():
```

Comment:
This test still requires ChatGPT API key, so I'd not include it into the tests. The best way would be to emulate streaming in the echo agent, but this is not that important. I'll merge with this test, but maybe we would prefer to redo this later.